### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Thebestzerospace**
`{ // Required "manifest_version": 3, "name": "My Extension", "version": "versionString", // Recommended "action": {...}, "default_locale": "en", "description": "A plain text description", "icons": {..import { useCallback, useEffect, useState } from "react"; import axios from "axios"; const useSearchUser = (name, perPage, curPage) => { const [isValid, setIsValid] = useState(undefined); const [loadi"}`

### CODE REVIEW

**Comment by anhanh92** [comment]
Tôi muốn xóa tất cả

**Review by anhanh92 — APPROVED** [review]

**Review by myldn — APPROVED** [review]
- [ ] 
Duplicate of #

**Comment by myldn** [comment]
> 
Thebestzerospace:patch-1

**Review by Bubbun — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ This cheat sheet is automatically generated from [GitHub Emoji API](https://api.
 - [Monkey Face](#monkey-face)
 - [Emotion](#emotion)
 
-#### Face Smiling
+#### Face Smiling£###
 
 | | ico | shortcode | ico | shortcode | |
```
Comment (Bubbun) [inline README.md:40]: gh pr checkout 594

**Review by mslovelymiaboo — COMMENTED** [review]
Please delete me and get my mo ey. A I from the party's please they took enough am not splitting the check

**Pull request closed without merging**